code/datums/elements/plasma_on_attack.dm

```diff
@@ -0,0 +1,17 @@
+/datum/element/plasma_on_attack
+	element_flags = ELEMENT_BESPOKE
+	id_arg_index = 2
+
+	var/damage_plasma_multiplier = 1
+
+/datum/element/plasma_on_attack/Attach(datum/target, damage_plasma_multiplier)
+	. = ..()
+	if(!isxeno(target))
+		return ELEMENT_INCOMPATIBLE
+	RegisterSignal(target, COMSIG_XENOMORPH_ATTACK_LIVING, .proc/damage_dealt)
+	src.damage_plasma_multiplier = damage_plasma_multiplier
```

Comment:
? Doesn't this mean that if one changes this number, everyone has it changed?

Reply:
Bespoke